src/server/routes/questions.ts

```diff
@@ -5,11 +5,11 @@ import question from '../mock/question';
 const router = express.Router();
 
 // GET questions
-router.get('/questions', (req: Request, res: Response) => {
-  const { unitId, unitType, questionCounts }: GetQuestionsBody = req.body;
+router.get('/questions', (req: Request<unknown, unknown, unknown, GetQuestionsBody>, res: Response) => {
+  const { unitId, unitType, questionCounts }: GetQuestionsBody = req.query;
   if (!unitId || !unitType || !questionCounts) {
     res.status(404);
-    return res.send({ errorMessage: 'Incorrect request body.' });
+    return res.send({ errorMessage: 'Incorrect request params.' });
```

Comment:
Params and queries are different things: https://stackoverflow.com/a/14508182
We should change it to `return res.send({ errorMessage: 'Incorrect request queries.' });` instead.

Reply:
Although it is received as queries in express, the client sends it as params.